integration/test_flow_integration.py

```diff
@@ -892,7 +933,15 @@ async def get_kv_item(full_path, key):
 
 
 def test_writing_int_key(setup_teardown_test):
-    table = Table(setup_teardown_test.table_name, setup_teardown_test.driver())
+    if setup_teardown_test.driver_name == "SQLDriver":
+        create_sql_table(
```

Comment:
the call to create_sql_table "spams" the pytest log with multiple lines - please see if you can remove / reduce:

`
INFO sqlalchemy.engine.Engine BEGIN (implicit)
2022-12-20 16:05:03,748 INFO sqlalchemy.engine.Engine PRAGMA main.table_info("ByvRKrMpEi")
2022-12-20 16:05:03,749 INFO sqlalchemy.engine.Engine [raw sql] ()
2022-12-20 16:05:03,750 INFO sqlalchemy.engine.Engine PRAGMA temp.table_info("ByvRKrMpEi")
2022-12-20 16:05:03,750 INFO sqlalchemy.engine.Engine [raw sql] ()
2022-12-20 16:05:03,752 INFO sqlalchemy.engine.Engine 
CREATE TABLE "ByvRKrMpEi" (
        num INTEGER NOT NULL, 
        color VARCHAR, 
        PRIMARY KEY (num)
)
2022-12-20 16:05:03,752 INFO sqlalchemy.engine.Engine [no key 0.00025s] ()
2022-12-20 16:05:03,986 INFO sqlalchemy.engine.Engine COMMIT
`